openqa-investigate

```diff
@@ -151,6 +161,29 @@ investigate() {
         echoerr "Job already has a clone, skipping investigation. Use the env variable 'force=true' to trigger investigation jobs"
         return 0
     fi
+
+    # skip parallel children (will be handled together with parallel parent)
+    # note: For this to work, this script needs to be executed for all job results (so we won't miss the investigation in case the parallel parent has passed).
+    parallel_parents=$(echo "$job_data" | runjq -r '(.job.parents["Parallel"] | length)') || return $?
+    [[ $parallel_parents != 0 ]] \
+        && echoerr "Skipping investigation of job $id: it will be handled together with its parallel parent" && return 0
```

Comment:
```suggestion
        && echoerr "Skipping investigation of job $id: Investigation will be done on the parallel parent" && return 0
```

to avoid the potentially ambiguous "it"-reference and making it a tad shorter